src/chainstate/burn/db/sortdb.rs

```diff
@@ -986,20 +986,26 @@ impl<'a> SortitionHandleTx<'a> {
                     return Ok(None);
                 }
 
-                let chosen_recipient = reward_set_vrf_seed.choose(
+                if OUTPUTS_PER_COMMIT != 2 {
```

Comment:
Is this necessary?  `OUTPUTS_PER_COMMIT` is a static constant, right?

Reply:
It's not strictly necessary, but because the implementation of the address choosing depends on `OUTPUTS_PER_COMMIT` to be equal to 2, I want this code to panic if we ever change that constant.